_data/navigation.yml

```diff
@@ -1,5 +1,7 @@
 main:
   - title: "Posts"
     url: /posts/
+  - title: "Leaderboard"
```

Comment:
Should probably name it ice leaderboard bc it's under the context of `bread-gang` which doesn't directly imply icing